chatty-cathy-client/src/main/java/org/client/Client.java

```diff
@@ -0,0 +1,70 @@
+package org.client;
+
+import org.common.Message;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.*;
+
+public class Client {
+
+    private static final Logger logger = LoggerFactory.getLogger(Client.class);
+    private ObjectInputStream serverIn = null;
+
+    public Client(String addr, int port) throws IOException {
+        BufferedReader in;
+        ObjectOutputStream out;
+
+        Socket s;
+        try {
+            s = new Socket(addr, port);
+            logger.info("Connected");
+            in = new BufferedReader(new InputStreamReader(System.in));
+            out = new ObjectOutputStream(s.getOutputStream());
+            serverIn = new ObjectInputStream(s.getInputStream());
+
+            Thread readThread = new Thread(() -> {
+                try {
+                    while (true) {
+                        Message message = (Message) serverIn.readObject();
+                        logger.info(message.getUsername() + ": " + message.getMessage());
+                    }
+                } catch (IOException | ClassNotFoundException e) {
+                    logger.error(String.valueOf(e));
+                }
+            });
+            readThread.start();
+        } catch (IOException i) {
+            logger.error(String.valueOf(i));
+            return;
+        }
+
+        String m = "";
```

Comment:
This variable is too important to be called m. Also it would be better if it was defined underneath the username, as that is the order they are used in Message's constructor.


```suggestion
        String userInput = "";
```